circle.yml

```diff
@@ -1,18 +1,71 @@
-dependencies: 
-  cache_directories: 
+# Setup in CircleCI account the following ENV variables:
+# BINTRAY_ACCOUNT
+# BINTRAY_API_KEY
+# DOCKER_USER
+# DOCKER_EMAIL
+# DOCKER_PASSWORD
+general:
+  branches:
+#    only:
+#      - master
+#      - /v[0-9]+(\.[0-9]+)*/
+  artifacts:
+    - /tmp/st2-packages
+
+machine:
+  # Overwrite these ENV variables in parametrized (manual/API) builds
+  environment:
+    ST2_GITURL: https://github.com/StackStorm/st2
+    ST2_GITREV: master
+    ST2_DOCKERFILES_REPO: https://github.com/armab/st2-dockerfiles
+    BUILD_DOCKER: 0
+    DEPLOY_DOCKER: 0
+    DEPLOY_PACKAGES: 0
+  pre:
+    # Paswordless SSH between parallel build nodes
+    - sudo cp /home/ubuntu/.ssh/config /root/.ssh/config
```

Comment:
`0` `1` `2` `3` machines in parallel matrix builds - they're different nodes that have connection between each other only via SSH.